src/plugins/PointerSizePlugin.js

```diff
@@ -0,0 +1,109 @@
+import { BasePlugin } from './BasePlugin';
+import { Slider } from '../ui-elements/Slider';
+
+/**
+ * @export
+ * @class PointerSizePlugin
+ * @extends {BasePlugin}
+ *
+ */
+export class PointerSizePlugin extends BasePlugin {
+  /**
+   *Creates an instance of PointerSizePlugin.
+   * @param {object} params
+   * @param {string | HTMLElement} [params.pointerSliders]
+   * @param {number} [params.defaultPointerSize=0.5]
+   * @memberof PointerSizePlugin
+   */
+  constructor({
```

Comment:
Again: let's refactor this constructor to be `new PointerSizePlugin(selector, { defaultValue })`